user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -114,6 +120,12 @@ public static UserOperatorConfig fromMap(Map<String, String> map) {
             reconciliationInterval = Long.parseLong(reconciliationIntervalEnvVar);
         }
 
+        int scramPasswordLength = DEFAULT_SCRAM_SHA_PASSWORD_LENGTH;
+        String scramPasswordLengthEnvVar = map.get(UserOperatorConfig.STRIMZI_SCRAM_SHA_PASSWORD_LENGTH);
+        if (scramPasswordLengthEnvVar != null) {
+            scramPasswordLength = Integer.parseInt(scramPasswordLengthEnvVar);
+        }
```

Comment:
I think this one could be elegantly be written by a ternary operator and remove unneeded variables what do you think? Something like:
```
scramPasswordLength =  map.get(UserOperatorConfig.STRIMZI_SCRAM_SHA_PASSWORD_LENGTH) != null ? Integer.parseInt(scramPasswordLengthEnvVar) : DEFAULT_SCRAM_SHA_PASSWORD_LENGTH;
```